### ISSUE TITLE
Could not pass event PlayerInteractEvent to Pueblos v2.1.2

### ISSUE THREAD

**Issue by strawmelonjuice** [SuperRonanCraft/Pueblos#3]
Hiya, the Pueblos plugin is working amazing, however my server logs are spammed with this error over and over and I was wondering if that was fixable, as reading the logs gets a little difficult now :)
```diff
- [14:39:40] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to Pueblos v2.1.2
java.lang.NullPointerException: Cannot invoke "me.RonanCraft.Pueblos.claims.enums.CLAIM_FLAG.getDefault()" because "flag" is null
	at me.RonanCraft.Pueblos.claims.data.Claim_FlagHandler.getFlag(Claim_FlagHandler.java:28) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.claims.ClaimHandler.allowInteract(ClaimHandler.java:285) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.player.events.PueblosEvents.allowInteract(PueblosEvents.java:37) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.player.events.EventInteract.onInteract(EventInteract.java:54) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.player.events.EventListener.onInteract(EventListener.java:62) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:git-Purpur-1842]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:678) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:545) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.level.ServerPlayerGameMode.useItemOn(ServerPlayerGameMode.java:528) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItemOn(ServerGamePacketListenerImpl.java:2051) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemOnPacket.handle(ServerboundUseItemOnPacket.java:37) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemOnPacket.a(ServerboundUseItemOnPacket.java:9) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:51) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1368) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:185) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1345) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1338) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:114) ~[?:?]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1472) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1194) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:310) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[14:39:40] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to Pueblos v2.1.2
java.lang.NullPointerException: Cannot invoke "me.RonanCraft.Pueblos.claims.enums.CLAIM_FLAG.getDefault()" because "flag" is null
	at me.RonanCraft.Pueblos.claims.data.Claim_FlagHandler.getFlag(Claim_FlagHandler.java:28) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.claims.ClaimHandler.allowInteract(ClaimHandler.java:285) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.player.events.PueblosEvents.allowInteract(PueblosEvents.java:37) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.player.events.EventInteract.onInteract(EventInteract.java:54) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at me.RonanCraft.Pueblos.player.events.EventListener.onInteract(EventListener.java:62) ~[Pueblos-LATEST-[2.1.2].jar:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:git-Purpur-1842]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:678) ~[purpur-api-1.19.2-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:545) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.level.ServerPlayerGameMode.useItemOn(ServerPlayerGameMode.java:528) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleUseItemOn(ServerGamePacketListenerImpl.java:2051) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemOnPacket.handle(ServerboundUseItemOnPacket.java:37) ~[?:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemOnPacket.a(ServerboundUseItemOnPacket.java:9) ~[?:?]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:51) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1368) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:185) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1345) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1338) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.util.thread.BlockableEventLoop.runAllTasks(BlockableEventLoop.java:114) ~[?:?]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1472) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1194) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:310) ~[purpur-1.19.2.jar:git-Purpur-1842]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```
It seems to occur when someone walks over a claim, but if I'm reading correct it has to do with reading the FLAGS of that claim, right?

I am running Purpur build 1835 1.19.2.

**Comment by strawmelonjuice** [comment]
@SuperRonanCraft would you be able to take a look at this?

**Comment by SuperRonanCraft** [comment]
This has been fixed in the latest 3.0.0 patch ;)
